handbook_reviewer_guidelines.md

```diff
@@ -0,0 +1,25 @@
+This document contains the guidelines for anyone reviewing content for the Operate First Community Handbook.
+
+== Styling content
+
+
+== Criteria for each column to progress to the next column:
+=== To enter ‘Proposed content’
+- [ ] An issue or PR is made and assigned to the ‘Community Handbook’ project
+- [ ] To enter “Content in progress” (decider = reviewer)
```

Comment:
Should line 9 be its own heading, rather than a box to check?

Reply:
Yep, thanks, fixed!